jekyll/_cci2/server/v4.3/installation/phase-2-core-services.adoc

```diff
@@ -679,6 +677,18 @@ object_storage:
     region: "<role-region>"
     irsaRole: "<irsa-arn>"
 ----
+
+**Non-presigned URLs (Optional)**
+If you wish to store artifacts larger than 5GB, you will need to disable presigned URLS. Add the following to the `object_storage.s3` section:
```

Comment:
It might be worth including something about what is actually happening when you run with presigned URLs disabled (i.e. that we perform a multi-part upload to S3), as I can see someone misreading this as just disabling the presigning of the URLs (i.e. disabling the auth), which it isn't (the security between the 2 is essentially equivalent).

May also be good to point out that disabling presigned URLs may also offer better resilience and throughput (i.e. faster uploads) compared with using presigned URLS, in addition to enabling uploads greater than 5GB.